test/integration/full/run-partial/after-method.js

```diff
@@ -0,0 +1,60 @@
+describe('run-partial', function() {
+  'use strict';
+  var ruleName = 'heading-order'
+  var runPartialRecursive = axe.testUtils.runPartialRecursive
+  var clone = axe.utils.clone
+  
+  beforeEach(function (done) {
+    axe.testUtils.awaitNestedLoad(done);
+  });
+  
+  it('can pass with a complex "after" method', function (done) {
+    var options = { runOnly: ruleName };
+    var context = { exclude: [] }
+    Promise.all(
+      runPartialRecursive(clone(context), options)
+    ).then(function (partialResults) {
+      return Promise.all([
+        axe.finishRun(partialResults, options),
+        axe.run(clone(context), options)
```

Comment:
Was confused why you were running `axe.run` here. Looking further into the test I see you're making sure they are the same result. Might be better if this sort of test was first defined like:

"it(should give the same results as `axe.run`)"